pipeline/metadata/5.1.yaml

```diff
@@ -414,8 +429,23 @@
     - ibmc
     - rgw
     - stage-3
-- name: "test-rgw-multisite-primary-to-secondary-tier-1-extd"
-  suite: "suites/pacific/rgw/tier-1-extn_rgw_multisite-primary-to-secondary.yaml"
+- name: "test-rgw-ms-omap-datalog-on-primary.yaml"
+  suite: "suites/pacific/rgw/tier-1_rgw_ms-omap-datalog-on-primary.yaml"
+  global-conf: "conf/pacific/rgw/rgw_multisite.yaml"
+  platform: "rhel-8"
+  rhbuild: "5.1"
+  inventory:
+    openstack: "conf/inventory/rhel-8-latest.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-8-latest.yaml"
+  metadata:
+    - sanity
+    - tier-2
+    - openstack
+    - ibmc
+    - rgw
+    - stage-3
```

Comment:
you can add stage-4 for tier-2  in 5.1.yaml  , as we are maintaining only 5 test suite  in a single stage.